helios/pipeViewer/scripts/alf_gen/ALFLayout.py

```diff
@@ -266,6 +266,7 @@ def __init__(self,
                      spacing,
                      margins,
                      include_detail_column : bool,
+                     content_width : int,
```

Comment:
To allow backward compatibility, can you swap the order of the new param with time_offset?